index.js

```diff
@@ -186,7 +209,9 @@ class KeyringController extends EventEmitter {
     if (!encryptedVault) {
       throw new Error('Cannot unlock without a previous vault.');
     }
-    await this.encryptor.decrypt(password, encryptedVault);
+
+    const result = await this.encryptor.decrypt(password, encryptedVault);
```

Comment:
(nit) the more common spelling seems to be encrypter :shrug:  
https://en.wiktionary.org/wiki/encryptor

Reply:
Agreed, but this is an existing practice that doesn't need to be changed in this PR.